app/controllers/pagination.rb

```diff
@@ -0,0 +1,20 @@
+module Pagination
```

Comment:
i really like that you created the pagination on your own AND that it's in a module. my only piece of feedback here would be to make these methods available to the module itself rather than having to include the module in the entire controller.

every method in the controller doesn't need access to this module, right? 

so if we make these method `self.on_page` and `self.total_pages`.. etc. - then, we could call them directly in the methods we need rather than having to include the module. 

let me know if this is clear - i'm finding it hard to use only text to explain this!